.env

```diff
@@ -0,0 +1 @@
+React_App_BackendBaseUrl="http://localhost:3002"
```

Comment:
I have an idea. Since it will take a lot of time to implement different querying options in the backend, let's remove the `.env` file here completely(since we will only have one option in machine selector i.e. localhost), we can use the default machine in `constants.ts`.

Reply:
@aquibbaig actually this is the backend url where we will hit all are API call that will be independent of the selected machine